network-api/networkapi/utility/templatetags/primary_active_nav.py

```diff
@@ -4,13 +4,12 @@
 
 
 @register.simple_tag(name='primary_active_nav')
-def primary_active_nav(request, target_path):
+def primary_active_nav(request, root_path, target_path):
 
     if not request:
         return ""
 
-    # remove langauge code segment from path to make path comparison easier
-    lang = f"/{request.LANGUAGE_CODE}"
-    path_info = request.path_info[len(lang):] if request.path_info.startswith(lang) else request.path_info
+    if target_path == root_path:
```

Comment:
What happened to the localization issue here? Does passing "root_path" do something about that? (If so, can you comment that somewhere around here?)

Reply:
both of these are now "the url for that page, for this user", so both of them will always contain the relevant locale code.